package.json

```diff
@@ -7,6 +7,9 @@
     "url": "https://github.com/MetaMask/metamask-extension.git"
   },
   "scripts": {
+    "webpack": "tsx ./development/webpack/launch.ts",
```

Comment:
Ok, so `webpack` isn't the greatest command name, and I'm sure it will change in the future, especially since I want to switch to `rspack` eventually.

Suggestions welcome!

Reply:
Doesn't matter much to me, since we can rename to whatever we want at any time later.